apps/web/components/SocialSharing/SocialSharing.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { FC } from 'react'
+import Head from 'next/head'
+
+interface SocialSharingProps {
+  title: string
+  description?: string
+  imageUrl?: string
+  imageWidth?: string
+  imageHeight?: string
+}
+
+export const SocialSharing: FC<SocialSharingProps> = ({
+  title,
+  description = 'Ísland.is er upplýsinga- og þjónustuveita opinberra aðila á Íslandi. Þar getur fólk og fyrirtæki fengið upplýsingar og notið margvíslegrar þjónustu hjá opinberum aðilum á einum stað í gegnum eina gátt.',
+  imageUrl = '/island-fb-1200x630.png',
```

Comment:
I believe you need to provide an absolute URL for the image, otherwise you will get `Provided og:image URL, /island-fb-1200x630.png was not a valid URL.`

Reply:
somehow thought i'd already merged this PR! I will fix this now 😃